examples/generative/md/real_nvp.md

```diff
@@ -850,9 +859,13 @@ axes[1, 1].set_ylim([-2, 2])
 
 ```
 </div>
-![png](/img/examples/generative/real_nvp/real_nvp_13_1.png)
+    
+![png](../examples/generative/md/real_nvp_files/../examples/generative/md/real_nvp_13_2.png)
```

Comment:
The image paths are incorrect -- they should be reverted to the previous format.